axelor-project/src/main/resources/views/Project.xml

```diff
@@ -370,16 +370,12 @@
 		<panel name="characteristicsPanel" title="Characteristics" colSpan="12"
 			sidebar="true" canCollapse="true">
 			<panel name="configPanel" colSpan="12">
-				<field if="__config__.app.isApp('business-project')" if-module="axelor-business-project"
-					name="isProject" colSpan="6" widget="boolean-switch" />
-				<field if="__config__.app.isApp('business-project')" if-module="axelor-business-project"
-					name="isBusinessProject" colSpan="6" widget="boolean-switch" />
+				<field name="isProject" colSpan="6" widget="boolean-switch" />
+				<field name="isBusinessProject" colSpan="6" widget="boolean-switch" />
```

Comment:
`isBusinessProject` is defined in `axelor-business-project` so I think that app/module checks should not be removed